scripts/postProcessDocs.js

```diff
@@ -0,0 +1,33 @@
+const jsonDoc = require('../docs.json');
+const _ = require('lodash');
+const fs = require('fs');
+
+const docRegEx = /https:\/\/doc.cognitedata.com\/api\/v1\/#operation\/([a-zA-Z0-9]+)/g;
+const header =
+`import { CogniteClient } from '@cognite/sdk';\n
```

Comment:
String templates should support multilines without adding `\n` 🤔 
https://stackoverflow.com/a/36268846/4462088

Reply:
yeah, I know, but I need 2 new lines, and that looks a bit messy
```
const header =
`import { CogniteClient } from '@cognite/sdk';

const client = new CogniteClient({ appId: '[APP NAME]' });
client.loginWithApiKey({
  project: '[PROJECT]',
  apiKey: '[API_KEY]'
});

`;
```

Reply:
especially in the bottom